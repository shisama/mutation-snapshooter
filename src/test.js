const Vue = require('vue');
const Vuex = require('vuex');
const mutationsSnapshooter = require('.');

Vue.use(Vuex);

const state = {
  counter: {
    value: 0
  }
}

const mutations = {
  increment: state => state.counter.value++,
  decrement: state => state.counter.value--
}

const actions = {
  increment: ({
    commit
  }) => commit('increment'),
  decrement: ({
    commit
  }) => commit('decrement')
}

const store = new Vuex.Store({
  state,
  mutations,
  actions
});

mutationsSnapshooter(state, mutations);