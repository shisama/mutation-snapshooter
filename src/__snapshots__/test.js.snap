// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`decrement 1`] = `
"Snapshot Diff:
- First value
+ Second value

  Object {
    \\"counter\\": Object {
-     \\"value\\": 0,
+     \\"value\\": -1,
    },
  }"
`;

exports[`increment 1`] = `
"Snapshot Diff:
- First value
+ Second value

  Object {
    \\"counter\\": Object {
-     \\"value\\": 0,
+     \\"value\\": 1,
    },
  }"
`;
